testing_gateway.go

```diff
@@ -0,0 +1,42 @@
+package braintree
+
+// This action is only available in the sandbox environment.
+type TestingGateway struct {
+	*TransactionGateway
+}
+
+// Settle settles a transaction.
+func (g *TestingGateway) Settle(id string) (*Transaction, error) {
+	return g.setStatus(id, "settle")
+}
+
+// Confirms a settlement.
+func (g *TestingGateway) SettlementConfirm(id string) (*Transaction, error) {
+	return g.setStatus(id, "settlement_confirm")
+}
+
+// Declined a settlements.
+func (g *TestingGateway) SettlementDecline(id string) (*Transaction, error) {
+	return g.setStatus(id, "settlement_decline")
+}
+
+// Declined a settlements.
```

Comment:
This comment and the two above it should follow the go pattern of function name as the first word.
e.g. `// SettlementPending changes the transaction's status to settlement_pending.`

Ref: https://blog.golang.org/godoc-documenting-go-code